modprobe.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Modprobe on start up
+
+[Service]
+ExecStart=/usr/sbin/modprobe libconfig
```

Comment:
shouldn't this be `libcomposite`?

Reply:
More than likely. I ended up selling my pi though so I can't exactly test if it works or not, however I'll assume that it is as I'm not the brightest bulb when it comes to these kinds of things.

Reply:
Ok `libconfig` failed for me and `libcomposite` failed
rpi400 ubuntu 64-bit, self compiled pi400kb

Reply:
Excellent, thank you very much. I'll change this asap!